examples/rpc/src/client.rs

```diff
@@ -28,22 +28,21 @@ impl ServerPuppet {
             CLIENT_PORT
         ).unwrap();
 
-        if let Some(tree) = owner.get_tree() {
-            let tree = unsafe { tree.assume_safe() };

```

Comment:
I have made these changes so that steps like this wouldn't just fail silently. This is Godot API operation so we don't have any other way to handle the error I think.